### DISCUSSION TITLE
Snipe-IT | SSO support

### DISCUSSION THREAD

**Discussion by raviprasanthg** [snipe/snipe-it#13357] [General]
Hi Folks,

We are using Snip-it with self-hosted subscription model, does this support SSO either with LDAP or Okta? Please help.

If this is not supporting SSO, what is the next best plan to purchase to support SSO (single sign-on)
<img width="1424" alt="image" src="https://github.com/snipe/snipe-it/assets/108456291/39f8ee4f-88d7-477b-bebf-0177a12491ee">

**Comment by snipe** [comment]
What we host for you in the paid option is identical to what you’re running yourself, so yes.

**Reply by raviprasanthg** [reply]
Thanks for the help in answering
